internalsdk/ios/config.go

```diff
@@ -100,8 +104,11 @@ type UserConfig struct {
 
 // TODO: Implement a timeout mechanism to handle prolonged execution times and potentially execute this method in the background to maintain smooth UI startup performance.
 func (cf *configurer) Configure(userID int, proToken string, refreshProxies bool) (*ConfigResult, error) {
+	// Log the full method run time.
+	defer func(start time.Time) {
```

Comment:
@myleshorton Also Running this in IOS, It still takes around, Is this expected?
`ios: config.go:109 Configured completed in 7.802894417s`

Reply:
The biggest thing is this method should definitely not block the UI thread

Reply:
> That's not super surprising, no. What was it taking before?
For me, it's the same as before,

Reply:
> The biggest thing is this method should definitely not block the UI thread

Yeah, We might need to make changes soon.

Reply:
Yeah in uncensored countries, there won't be much difference. We should, however, look at what is actually taking that time -- I'll eventually trace all this stuff.

Reply:
#1259 This should fix IOS crash and you should be able to run it and see whats going on